src/win/fs.c

```diff
@@ -142,6 +142,57 @@ void uv_fs_init(void) {
 }
 
 
+static unsigned int fs__decode_wtf8_char(const char** input) {
+  unsigned int code_point;
+  unsigned char b = **input, b2, b3, b4;
+  if (b >= 0x00 && b <= 0x7F) {
```

Comment:
```suggestion
  if (b <= 0x7F) {
```
(b is unsigned, so this would generate a compiler warning)